src/Symfony/Component/Security/Http/Firewall/UsernamePasswordFormAuthenticationListener.php

```diff
@@ -95,6 +95,10 @@ protected function attemptAuthentication(Request $request)
             throw new BadCredentialsException('Invalid username.');
         }
 
+        if (null === $password) {
+            throw new \LogicException(sprintf('The key "%s" cannot be null.', $this->options['password_parameter']));
```

Comment:
Maybe try to explain the problem?

`sprintf('The key password cannot be null; check that the password field name of the form matches "%s".', $this->options['password_parameter'], $this->options['password_parameter']);`